saiph/reduction/famd.py

```diff
@@ -325,6 +325,8 @@ def _compute_contributions(
     # compute the contribution
     raw_contributions = (U * s) ** 2 / eig
     raw_contributions = raw_contributions * model.column_weights[:, np.newaxis]
+    summed_contributions: NDArray[np.float_] = np.array(raw_contributions.sum(axis=0))
+    raw_contributions /= summed_contributions
```

Comment:
Did you try to do as we are doing in PCA? 
```python
correlations = get_variable_correlation(model, df)
cos2 = correlations**2
contributions = cos2.div(cos2.sum(axis=0), axis=1).mul(100)
```

Otherwise, I'm ok like this